go.mod

```diff
@@ -29,68 +29,68 @@ require (
+	github.com/grafana/e2e v0.1.1-0.20221018202458-cffd2bb71c7b
 	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645
-	github.com/hashicorp/go-hclog v1.2.1
+	github.com/hashicorp/go-hclog v1.2.2
 	github.com/hashicorp/go-plugin v1.4.4
-	github.com/jaegertracing/jaeger v1.36.0
+	github.com/jaegertracing/jaeger v1.37.0
 	github.com/jedib0t/go-pretty/v6 v6.2.4
 	github.com/json-iterator/go v1.1.12
 	github.com/jsternberg/zap-logfmt v1.2.0
-	github.com/klauspost/compress v1.15.9
-	github.com/minio/minio-go/v7 v7.0.16-0.20211116163909-d00629356463
+	github.com/klauspost/compress v1.15.11
+	github.com/minio/minio-go/v7 v7.0.37
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/olekukonko/tablewriter v0.0.5
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.57.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter v0.57.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.57.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.57.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver v0.57.2
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.60.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/zipkinexporter v0.51.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger v0.60.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.60.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver v0.51.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver v0.57.2
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.57.2
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.60.0
 	github.com/opentracing-contrib/go-grpc v0.0.0-20210225150812-73cb765af46e
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pierrec/lz4/v4 v4.1.15
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.13.0
-	github.com/prometheus/client_model v0.2.0
+	github.com/prometheus/client_model v0.3.0
 	github.com/prometheus/common v0.37.0
 	github.com/prometheus/prometheus v1.8.2-0.20221021121301-51a44e6657c3
-	github.com/prometheus/statsd_exporter v0.21.0 // indirect
+	github.com/prometheus/statsd_exporter v0.22.7 // indirect
 	github.com/segmentio/fasthash v0.0.0-20180216231524-a72b379d632e
 	github.com/segmentio/parquet-go v0.0.0-20221019213850-8b6e2b21a101
-	github.com/sirupsen/logrus v1.8.1
-	github.com/sony/gobreaker v0.4.1
+	github.com/sirupsen/logrus v1.9.0
+	github.com/sony/gobreaker v0.5.0
 	github.com/spf13/viper v1.12.0
-	github.com/stretchr/testify v1.8.0
-	github.com/thanos-io/thanos v0.24.0
+	github.com/stretchr/testify v1.8.1
+	github.com/thanos-io/thanos v0.29.0-rc.0
```

Comment:
This is a big one.

Reply:
How do we feel about an rc here?  We could wait, or take the rc and quickly update when 29.0 comes.

Reply:
What is the motivation for upgrading to `0.29.0-rc.0` vs the latest stable which I assume is `0.28.0`?

Reply:
There was a recent breaking change in one of the dependencies that a bunch of projects share.

https://github.com/thanos-io/thanos/blob/v0.28.1/go.mod#L19

This caused a bunch of headache for updates, which has been updated in 29.0-rc.0 to point to the new location.